### PR TITLE
fix lintOnFly and wrong range(line) report

### PR DESCRIPTION
**Pull request by Project0**
fix lintonfly #27. Pipe current text to ruby processor.
Correct wrong line of error.

### CODE REVIEW

**Comment by mneumark** [comment]
Glad to see someone addressing this! Thanks!

File: lib/main.js [hunk 1]
```diff
@@ -39,21 +38,18 @@ export default {
...
               return;
             }
             toReturn.push({
-              range: helpers.rangeFromLineNumber(activeEditor, Number.parseInt(matches[1])),
-              type: "Error",
+              range: helpers.rangeFromLineNumber(activeEditor, Number.parseInt((matches[1] - 1))),
+              type: 'Error',
               text: matches[2],
-              filePath: file
+              filePath: activeEditor.getPath()
```
Comment (steelbrain) [inline lib/main.js]: can you store this in a variable instead of calling a function for each message?

**Comment by k2b6s9j** [comment]
Okay.

**Comment by k2b6s9j** [comment]
Next patch release.

**Pull request merged**